### ISSUE TITLE
Close button appears outside of dismissible alert box

### ISSUE THREAD

**Issue by vizui** [USPTO/designpatterns#95]
Applying the .alert-dismissible class to an alert causes the close button to be pushed outside of it's DIV container.

![image](https://cloud.githubusercontent.com/assets/3170948/11255077/40e5b6e6-8e12-11e5-8809-19e72a61ed64.png)

````html
<div class="alert alert-info alert-dismissible">
  <button class="close" type="button"><span aria-hidden="true">×</span><span class="sr-only">Close</span></button>
  <i class="alert-icon alert-icon-info"></i> Press the <strong>'A'</strong> key to automatically assign a case.
</div>
````

Reference: 
http://getbootstrap.com/components/#alerts-dismissible
https://uspto.github.io/designpatterns/1.x/docs/components/alerts.html

**Comment by scottmerker** [comment]
Was this resolved?  I see this working correctly right now in the library.

**Comment by AaronUXD** [comment]
Thought it was still a problem as recently as yesterday, but looks fine now to me too.

------------------------------------
aaron.pepe@uspto.gov
(571) 272-7185 | MDW 4D30

**Comment by AaronUXD** [comment]
See it again now, odd...

**Comment by scottmerker** [comment]
Fixed in version 1.4.1.
